docs/api/apiv3/components/schemas/storage_model.yml

```diff
@@ -51,19 +51,33 @@ properties:
         allOf:
           - $ref: './link.yml'
           - description: |-
-              Web uri of the storage instance
+              Web uri of the storage instance.
 
               **Resource**: N/A
       authorizationState:
         allOf:
-          - $ref: "./link.yml"
+          - $ref: './link.yml'
           - description: |-
               The urn of the storage connection state. Can be one of:
               
               - urn:openproject-org:api:v3:storages:authorization:Connected
               - urn:openproject-org:api:v3:storages:authorization:FailedAuthorization
               - urn:openproject-org:api:v3:storages:authorization:Error
               
+              **Resource**: N/A
+      authorize:
+        allOf:
+          - $ref: './link.yml'
+          - description: |-
+              The link to the starting point of the authorization cycle for a configured storage provider.
+              
+              # Conditions
+              
+              `authorizationState` is on of:
+              
+              - urn:openproject-org:api:v3:storages:authorization:FailedAuthorization
+              - urn:openproject-org:api:v3:storages:authorization:Error
```

Comment:
@apfohl I disagree: If the authorizationState is Error then there is nothing the user can do, apart from reloading the page. Telling the user to reauthorize when the connection is bad is misleading.

Reply:
This is true we can leave it out in this case as well.